src/extension/background-script/actions/nostr/removePrivateKey.ts

```diff
@@ -0,0 +1,27 @@
+import type { MessagePrivateKeyRemove } from "~/types";
+
+import state from "../../state";
+
+const removePrivateKey = async (message: MessagePrivateKeyRemove) => {
+  const id = message.args?.id || state.getState().currentAccountId;
+
+  const accounts = state.getState().accounts;
+
+  if (id && Object.keys(accounts).includes(id)) {
+    const account = accounts[id];
+    if (account.nostrPrivateKey) delete account.nostrPrivateKey;
+    accounts[id] = account;
+    state.setState({ accounts });
+    await state.getState().saveToStorage();
+    return {
+      data: {
+        accountId: id,
```

Comment:
Why do we return the `accountId` as a result here? That isn't used anywhere, is it?

Reply:
Maybe it can be useful for some to get the related id because it could be either the provided id or the current one? Just a thought.

Reply:
But yeah, currently there's not need for it.